### ISSUE TITLE
[Enhancement] Implementation

### ISSUE THREAD

**Issue by dschier-wtd** [whiletruedoio/kube.while-true-do.io#2]
## Description

As a product owner, I want to have a setup according to the Architecture ( #1 ), and some documentation for developers and maintainers, so it can be used for testing purposes and production

## Criteria

- rollout must be non-interactive
- single cluster server instance
- all steps can be reproduced in a local setup
- configuration code and scripts are provided
- minimum requirements (for testing) are provided

## Additional information

- basically a plain "code-only" deployment of k3s
- reference: https://community.exoscale.com/documentation/compute/custom-templates/
- reference: https://github.com/rancher/k3os#automated-installation
- https://thepracticalsysadmin.com/build-a-pine64-kubernetes-cluster-with-k3os/
